CMakeLists.txt

```diff
@@ -670,12 +671,12 @@ target_include_directories(${OFFSETS_LIB} PRIVATE
   kernel/include
   ${ARCH_DIR}/${ARCH}/include
   )
-target_link_libraries(${OFFSETS_LIB} zephyr_interface)
-add_dependencies(     ${OFFSETS_LIB}
+add_dependencies(zephyr_interface
   ${SYSCALL_LIST_H_TARGET}
   ${DRIVER_VALIDATION_H_TARGET}
   ${KOBJ_TYPES_H_TARGET}
   )
+target_link_libraries(${OFFSETS_LIB} zephyr_interface)
```

Comment:
No need to move this line.
Can stay together with the other CMake calls handling `${OFFSETS_LIB}`

Reply:
Reverted so that line is not moved.